pages/sequential.jsx

```diff
@@ -78,9 +79,14 @@ module.exports = React.createClass({
     if (!this.state.showCvcHint) {
       cvcHintClassName += " hidden";
     }
+    // FIXME: Right now we only display this header for `en` locales
+    // this should be change to display for all once we have other locales translated.
+    if (/^(en)(\b|$)/.test(this.props.locales[0])) {
+      header = (<Header alt={this.getIntlMessage('donate_to_mozilla')}/>);
+    }
     return (
       <div className={className}>
-        <Header/>
+        {header}
```

Comment:
Nope because it will still be display on Google search result and I think letting Google pick up the next possible translated strings which could be the word "Donate now" is better than getting the whole `h1` which might not be translated at all.